README.md

```diff
@@ -5,15 +5,14 @@
 1. Select latest release (tag) that is less than or equal to a version of the cluster you plan to use CLI with.
 2. Download image from docker repository `docker pull vastdataorg/vast-db-cli:4.7.0-sp3`
 3. Download `vast-db-cli` from this repository.
-4. Move `vast-db-cli` into the `/bin/` directory.
-5. Make `vast-db-cli` executable: `chmod +x /bin/vast-db-cli`
-6. Execute the script.
+4. Make `vast-db-cli` executable: `chmod +x vast-db-cli`
+5. Configure it: `./vast-db-cli configure`
```

Comment:
Why do we change "Execute the script." to "Configure it: `./vast-db-cli configure`"? It's arguable whether it's clearer or not. Why do we ask to configure it instead of running `--help`?

Reply:
The first thing they will need to do is configure it -- lets just tell them that instead of making them figure it out.